tests/Provider/CSSClassProvider.php

```
@@ -376,6 +376,37 @@ public function __toString(): string
+                        public function __toString(): string
+                        {
+                            return 'class-one';
+                        }
+                    },
+                ],
+                [['classes' => 'class-two']],
+                ['class' => 'class-one class-two'],
+                'Should merge new classes into an existing Stringable class value.',
+            ],
+            'existing class as unsupported object' => [
+                ['class' => new \stdClass()],
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '405', column '33'). (undefined)

(MissingImport)